ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/BlockHeader.java

```diff
@@ -121,6 +166,31 @@ public Hash getHash() {
     return hash.get();
   }
 
+  public LogsBloomFilter getLogsBloom(final boolean forFilter) {
```

Comment:
would it be clearer to name this param forFilterPrivateLogs if it only applies when there's private logs?

Reply:
changed it to "addPrivateLoom".